src/app/util/basic-types.h

```diff
@@ -34,4 +34,7 @@ typedef uint16_t ClusterId;
 typedef uint16_t AttributeId;
 typedef uint16_t GroupId;
 typedef uint8_t CommandId;
+typedef uint16_t EventId;
+typedef uint64_t NodeId;
```

Comment:
No, this is wrong.  This header already defines `NodeId`.

Reply:
thanks for catching up. Add this PR to address it. 
https://github.com/project-chip/connectedhomeip/pull/3956